### PR TITLE
feat: add polish translations

### PR DESCRIPTION
**Pull request by lucasrabiec**
## Description

I translated strings to the polish language based on `en.json` file.

- [x] I have read and understand the CONTRIBUTING.md document in this repository

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by lucasrabiec** [comment]
@klapec could you take a look if this translation makes sense? Feel free to suggest changes.

**Review by DanRibbens — APPROVED** [review]
Great work, we really appreciate your help!
Let me know if I should remove the extra characters or not.

File: src/translations/pl.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    "accountAlreadyActivated": "To konto zostało już aktywowane.",
+    "autosaving": "Wystąpił problem podczas automatycznego zapisywania tego dokumentu.",
+    "correctInvalidFields": "Popraw nieprawidłowe pola.",
+    "deletingTitle": "Wystąpił błąd podczas usuwania {{title}}. Proszę, sprawdź swoje połączenie i spróbuj ponownie.",
+    "emailOrPasswordIncorrect": "Podany adres e-mail lub hasło jest nieprawidłowe.",
+    "followingFieldsInvalid_many": "Następujące pola są nieprawidłowe:",
+    "followingFieldsInvalid_one": "To pole jest nieprawidłowe:",
+    "forbidden": "Nie możesz wykonać tej akcji.",
+    "incorrectCollection": "Nieprawidłowa kolekcja",
+    "loadingDocument": "Wystapił problem podczas ładowania dokumentu o ID {{id}}.",
```
Comment (klapec) [inline src/translations/pl.json:61]: Typo, should be `Wystąpił`

File: src/translations/pl.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    "missingRequiredData": "Brak wymaganych danych.",
+    "noFilesUploaded": "Nie przesłano żadnych plików.",
+    "noMatchedField": "Nie znaleziono pasującego pola dla \"{{label}}\"",
+    "noUser": "Brak użytkownika",
+    "notFound": "Żądany zasób nie został znaleziony.",
+    "problemUploadingFile": "Wystąpił problem podczas przesyłania pliku.",
+    "tokenInvalidOrExpired": "Token jest nieprawidłowy lub wygasł.",
+    "unPublishingDocument": "Wystąpił problem podczas cofania publikacji tego dokumentu.",
+    "unauthorized": "Brak dostępu, musisz być zalogowany.",
+    "unknown": "Wystapił nieznany błąd.",
```
Comment (klapec) [inline src/translations/pl.json:74]: Typo, should be `Wystąpił`

File: src/translations/pl.json [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    "confirmRevertToSaved": "Potwierdź powrót do zapisanego",
+    "confirmUnpublish": "Potwierdź cofnięcie publikacji",
+    "confirmVersionRestoration": "Potwierdź przywrócenie wersji",
+    "currentDocumentStatus": "Bieżący status {{docStatus}} dokumentu",
+    "draft": "Szkic",
+    "lastSavedAgo": "Ostatnio zapisano {{distance, relativetime(minutes)}}",
+    "noFurtherVersionsFound": "Nie znaleziono dalszych wersji",
+    "noRowsFound": "Nie znaleziono {{label}}",
+    "preview": "Podgląd",
+    "problemRestoringVersion": "Wystapił problem podczas przywracania tej wersji",
```
Comment (klapec) [inline src/translations/pl.json:230]: Typo, should be `Wystąpił`

File: src/translations/pl.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    "addNew": "Dodaj nowy",
+    "addNewLabel": "Dodaj nowy {{label}}",
+    "addRelationship": "Dodaj relacje",
+    "blockType": "Typ Bloku",
+    "chooseFromExisting": "Wybierz z istniejących",
+    "chooseLabel": "Wybierz {{label}}",
+    "collapseAll": "Zwiń wszystko",
+    "editLabelData": "Edytuj dane {{label}}",
+    "fieldHasNo": "To pole nie posiada {{label}}",
+    "fieldRequiresRows": "To pole wymaga conajmniej {{label}}",
```
Comment (klapec) [inline src/translations/pl.json:92]: `co najmniej`

File: src/translations/pl.json [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+    "atLeastOneRow": "To pole wymaga co najmniej jednego wiersza.",
+    "emailAddress": "Wprowadź poprawny adres email.",
+    "enterNumber": "Wprowadź poprawny numer telefonu.",
+    "greaterThanMax": "Wartość \"{{value}}\" przekracza maksymalną, dozwoloną liczbę {{max}} znaków.",
+    "invalidInput": "To pole zawiera nieprawidłowe dane.",
+    "invalidSelection": "To pole ma nieprawidłowy wybór.",
+    "invalidSelections": "'To pole zawiera następujące, nieprawidłowe wybory:'",
+    "lessThanMin": "Wartość \"{{value}}\" jest mniejsza niż dozwolona, minimalna wartość: {{min}}.",
+    "longerThanMin": "Ta wartość musi być dłuższa niż minimalna długość znaków: {{minLength}}.",
+    "maxRows": "To pole może posiadać conajmniej {{maxRows}} wiersz(y).",
```
Comment (klapec) [inline src/translations/pl.json:202]: `co najmniej`

File: src/translations/pl.json [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+    "selectFile": "Wybierz plik"
+  },
+  "validation": {
+    "atLeastOneRow": "To pole wymaga co najmniej jednego wiersza.",
+    "emailAddress": "Wprowadź poprawny adres email.",
+    "enterNumber": "Wprowadź poprawny numer telefonu.",
+    "greaterThanMax": "Wartość \"{{value}}\" przekracza maksymalną, dozwoloną liczbę {{max}} znaków.",
+    "invalidInput": "To pole zawiera nieprawidłowe dane.",
+    "invalidSelection": "To pole ma nieprawidłowy wybór.",
+    "invalidSelections": "'To pole zawiera następujące, nieprawidłowe wybory:'",
```
[the remark this answers is not in the document]
Comment (klapec) [reply src/translations/pl.json:199]: I guess not

File: src/translations/pl.json [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+    "lessThanMin": "Wartość \"{{value}}\" jest mniejsza niż dozwolona, minimalna wartość: {{min}}.",
+    "longerThanMin": "Ta wartość musi być dłuższa niż minimalna długość znaków: {{minLength}}.",
+    "maxRows": "To pole może posiadać conajmniej {{maxRows}} wiersz(y).",
+    "minRows": "To pole musi posiadać conajmniej {{minRows}} wiersz(y).",
```
Comment (klapec) [inline src/translations/pl.json:203]: `co najmniej`

File: src/translations/pl.json [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+    "notValidDate": "\"{{value}}\" nie jest prawidłową datą.",
+    "required": "To pole jest wymagane.",
+    "requiresOneRow": "To pole wymaga conajmniej jednego wiersza.",
```
Comment (klapec) [inline src/translations/pl.json:206]: `co najmniej`

File: src/translations/pl.json [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+    "versionCount_many": "Znalezionych wersji: {{count}}",
+    "versionCount_none": "Nie znaleziono wersji",
+    "versionCount_one": "Znaleziono {{count}} wersję",
+    "versionCreatedOn": "Wersja {{version}} utworzona:",
+    "versionID": "ID wersji",
+    "versions": "Wersje",
+    "viewingVersion": "Przeglądanie wersji dla {{entityLabel}} {{documentTitle}}",
+    "viewingVersionGlobal": "`Przeglądanie wersji dla globalnej kolekcji {{entityLabel}}",
+    "viewingVersions": "Przeglądanie wersji {{entityLabel}} {{documentTitle}}",
+    "viewingVersionsGlobal": "`Przeglądanie ersji dla globalnej kolekcji {{entityLabel}}"
```
[the remark this answers is not in the document]
Comment (klapec) [reply src/translations/pl.json:255]: Definitely not - and there's a typo in here: `ersji` -> `wersji`

**Review by klapec — COMMENTED** [review]
Couple of typos sneaked in

**Comment by klapec** [comment]
Incredible work @lucasrabiec! There are a couple of typos left though - would you mind correcting them?

File: src/translations/pl.json (hunk 4 above)
Comment (lucasrabiec) [reply src/translations/pl.json:92]: Shame on me. My polish teacher would be disappointed. Luckily she doesn't have an account on GH. :D

File: src/translations/pl.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (lucasrabiec) [reply src/translations/pl.json:199]: Definitely not

**Comment by lucasrabiec** [comment]
@klapec thank you for CR, good points.
@DanRibbens I added adjustments here: https://github.com/lucasrabiec/payload/commit/548d57a537f7ad572ec46d8d9ae3c1defb343ab0

**Pull request merged**